src/BayesFilters/include/BayesFilters/sigma_point.h

```diff
@@ -12,12 +18,39 @@ namespace bfl
 {
 namespace sigma_point
 {
+    using FunctionEvaluation = std::function<std::pair<bool, bfl::Data>(const Eigen::Ref<const Eigen::MatrixXf>&)>;
+    
+    struct UTWeight
+    {
+        Eigen::VectorXd mean;
+        Eigen::VectorXd cov;
```

Comment:
Please use extended names: `covariance` instead of `cov` 😄

Reply:
Ok, I know this is a bad habit! :grin: